Client/Frontend/Browser/BrowserViewController/BrowserViewController+KeyCommands.swift

```diff
@@ -116,6 +116,21 @@ extension BrowserViewController {
         tabManager.removeTab(currentTab)
     }
 
+    @objc private func showTabTrayKeyCommand() {
+        TelemetryWrapper.recordEvent(category: .action, method: .press, object: .keyCommand, extras: ["action": "show-tab-tray"])
```

Comment:
Nit: formatting! :P

I want to make a suggestion, but I'm scared of formatting it in GitHub. lolololo

Reply:
@electricRGB All the `TelemetryWrapper.recordEvent` code in this file is on one line. Are you suggesting to put them all on multiple lines like this?
```
        TelemetryWrapper.recordEvent(category: .action,
                                     method: .press,
                                     object: .keyCommand,
                                     extras: ["action": "show-tab-tray"])
```

Reply:
Not all of them, no. But anything new we add, I would like to get in the habit of doing so, yeah.

Reply:
Well there's only a couple of them. I'll change them too since might as well be consistent in that file